code/style.css

```diff
@@ -1,4 +1,236 @@
 html {
-  background: #0025ff;
-  color: #fff;
+
 }
+
+body {
+margin: 0px;
+width: 100%;
+}
+
+.webpage {
+  margin: 15px;
+}
+
+/* HEADER */
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 10px;
+  margin: 15px;
+}
+
+.logo {
+  width: 50px;
+}
+
+.navigation {
+  display: flex;
+}
+
+.navigation-item {
+  margin-right: 10px;
+}
+
+.navigation.item: last-child {
+  margin-right: right 0;
+}
+
+.mobile-menu {
+  display: block;
+  margin: 4px;
+  width: 24px;
+  height: 20px;
+}
+
+.desktop-menu {
+  display: none;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+
+/* BIG NEWS CONTENT */
+
+.big-news-container {
+  width: 100%;
+  justify-content: center;
+}
+
+.big-news-image {
+  width: 100%;
+}
+
+.big-news-text {
+  width: 100%;
+  margin-left: 15px;
```

Comment:
This margin causes a horizontal scroll in mobile which we want to avoid when working with responsive web design. Either remove the margin or the 100% width to get rid of it :)

<img width="372" alt="Skärmavbild 2021-02-02 kl  10 35 53" src="https://user-images.githubusercontent.com/51529201/106580700-6f086480-6542-11eb-8f51-7b00686f95d5.png">